CONTRIBUTING.md

```diff
@@ -20,6 +22,26 @@ To build ICU4X, you will need the following dependencies:
 
 Certain tests may need further dependencies, these are documented below in the [Testing](#testing) section.
 
+### IDE setup
+
+ICU4X can be edited using any text editor capable of editing Rust code.
+
+#### Visual Studio Code
+
+Many ICU4X engineers use [Visual Studio Code](https://code.visualstudio.com/) with the [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer) extension.
+
+To build all code paths and improve build times in VSCode, we recommend the following settings. (Note: The second setting causes VSCode to build ICU4X with only the `und` locale, which reduces build times but also makes some tests fail; to run them normally, run `cargo test --all-features` on the command line.) To add these settings, choose "Preferences: Open Workspace Settings (JSON)" from the command palette (Ctrl+Shift+P):
+
+```javascript
```

Comment:
this isn't formatting correctly, you probably need
```suggestion
```json
```

Reply:
I can't have comments in JSON; they appear red highlighted. I don't see what's wrong with the JavaScript formatting.

Reply:
Well it's all just blue, JSON formatting highlights keys and values: https://github.com/unicode-org/icu4x/blob/7a7e93cf8790968970ecef7adb1610958f485e81/ffi/npm/package.json#L4

Reply:
Let's see how it looks in #4560